distro/data/mysql/create_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Apply all scripts found in docker-entrypoint-initdb.d/db/ directory.
+#
+# Similar workaround as for the PostgreSQL Docker image not running files from sub-dirs.
+# https://github.com/docker-library/postgres/issues/605#issuecomment-567236795
+#
+#
+set -e
+directory=/docker-entrypoint-initdb.d/db/
+if [ -d "${directory}" ]
+then
+    find ${directory}* -type f -execdir ls {} \; -execdir bash {} \;
+else
+    echo "Directory '${directory}' does not exist. No database to create. Exit 0."
+fi
+exit 0
```

Comment:
What happens when a service involving creation of its db is added  upon the nth startup? Is this script and therefore the underlying directory always going to be executed/searched through for new scripts that create new databases?

Reply:
> Is this script and therefore the underlying directory always going to be executed

It is not. This script is run only once and therefore there is no enhancement on this front from the previous behavior.